File: src/main/java/upbrella/be/user/dto/response/UmbrellaBorrowedByUserResponse.java
package upbrella.be.user.dto.response;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class UmbrellaBorrowedByUserResponse {

    private long uuid;
    private int elapsedDay;

    public static UmbrellaBorrowedByUserResponse of(long uuid, int elapsedDay) {

        return UmbrellaBorrowedByUserResponse.builder()
                .uuid(uuid)
                .elapsedDay(elapsedDay)
                .build();
    }
}

File: src/main/kotlin/upbrella/be/user/dto/response/UmbrellaBorrowedByUserResponse.kt
package upbrella.be.user.dto.response

data class UmbrellaBorrowedByUserResponse(
    val uuid: Long,
    val elapsedDay: Int
) {
    companion object {
        fun of(uuid: Long, elapsedDay: Int): UmbrellaBorrowedByUserResponse {
            return UmbrellaBorrowedByUserResponse(
                uuid = uuid,
                elapsedDay = elapsedDay
            )
        }
    }
}
